code/modules/admin/verbs/modifyvariables.dm

```diff
@@ -256,8 +256,8 @@ var/list/VVpixelmovement = list("step_x", "step_y", "bound_height", "bound_width
 			.["type"] = /list
 
 /client/proc/vv_parse_text(O, new_var)
-	if(O && findtext(new_var,"\["))
-		var/process_vars = alert(usr,"\[] detected in string, process as variables?","Process Variables?","Yes","No")
+	if(O && findtext(new_var, "\["))
+		var/process_vars = alert(usr, "\[] detected in string, process as variables?", "Process Variables?", "Yes", "No")

```

Comment:
An escape on the `[` means that rather than treating `[]` as an embedded expression, it treats it as a text literal- it works, and the formatting is correct as well.